src/taipy/core/data/_data_manager.py

```diff
@@ -101,35 +104,46 @@ def __create(
             raise InvalidDataNodeType(data_node_config.storage_type)
 
     @classmethod
-    def _clean_pickle_file(cls, data_node: Union[DataNode, DataNodeId]):
-        data_node = cls._get(data_node) if isinstance(data_node, str) else data_node
+    def _clean_pickle_file(cls, data_node: DataNode):
```

Comment:
What is this signature change about?

Reply:
Ah I was thinking about merging clean pickle files and updating the preserve file so that we don't have to load the entity twice, once for cleaning pickle files and once again for preserving the file path. Though like you said in your next comment, they are unrelated features.